module/type/TypeBrowser.kt

```diff
@@ -98,7 +98,7 @@ class TypeBrowser(isOpen: Boolean = false, order: Int) : Browsers.Browser(isOpen
     private fun exportButton(navState: Navigator.NavigatorState<ThingTypeState<*, *>>) = IconButtonArg(
         icon = Icon.EXPORT,
         enabled = Service.project.current != null && !Service.schema.hasRunningCommand,
-        tooltip = Tooltip.Arg(title = Label.EXPORT_SCHEMA)
+        tooltip = Tooltip.Arg(title = Label.EXPORT_TYPES)
```

Comment:
Strictly speaking, our existing export button on the Type Browser only exports the types, not the whole schema (including rules).